iosApp/Survey/Sources/Supports/Extensions/MokoResources/StringResource+Localized.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  StringResource+Localized.swift
+//  Survey
+//
+//  Created by Bliss on 18/10/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Shared
+
+extension String {
+
+    static let localizeId = MR.strings()
+}
+
+extension StringResource {
+
+    var localized: String {
```

Comment:
Just curious, can we have `callAsFunction` instead?
```swift
extension StringResource {

	func callAsFunction() {
		desc().localized()
	}
}

// then
String.localizeId.reset_password_button_reset()
```

Reply:
Nice suggestion 🚀